src/backend/distributed/commands/distribute_object_ops.c

```diff
@@ -1396,6 +1417,11 @@ GetDistributeObjectOps(Node *node)
 			return &Any_Index;
 		}
 
+		case T_ViewStmt:
```

Comment:
Mat views also  use `T_ViewStmt` and `OBJECT_VIEW`. I could not spot any gap that we'd use this code for mat. view, but would be good to double check by you as well that mat views cannot sneak in any of the code paths